src/ontology/mondo-edit.obo

```diff
@@ -71,6 +71,14 @@ property_value: IAO:0000700 MONDO:0021125
 property_value: IAO:0000700 MONDO:0021178
 property_value: IAO:0000700 MONDO:0042489
 
+[Term]
+id: MONDO:0000000
+name: TRIP11 related skeletal dysplasia
+def: "TRIP11 related skeletal dysplasia describes both achondrogenesis type 1a and ondontochondrodysplasia. Reduced protein function in TRIP11 causes a spectrum of skeletal symptoms from a more mild phenotype, called ondontochondrodysplasia, with features including short stature and joint laxity to a more severe phenotype, called achondrogenesis type 1a, which presents as a lethal prenatal or neonatal skeletal dysplasia. The clinical severity of achondrogenesis compared to ondontochondrodysplasia is related to the residual function of the gene which is not currently possible to anticipate based on genotype alone. The phenotype cannot fully be predicted by genotype alone, evidenced by variants being reported with both phenotypes." [PMID:20089971, PMID:30728324, PMID:31903676]
+is_a: MONDO:0018230 ! skeletal dysplasia
+property_value: http://purl.org/dc/elements/1.1/date "2024-06-27T14:50:03Z" xsd:dateTime
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-0587-4693
+
```

Comment:
please add the term tracker item (link to the GitHub URL for this term):
https://mondo.readthedocs.io/en/latest/editors-guide/f-entities/#term-tracker-item